Decentralization - Using Pledge as a Bidding Parameter to Determine Saturation Limit.md

```diff
@@ -0,0 +1,233 @@
+CIP: TBD\
+Title: Decentralization: Using Pledge as a Bidding Parameter to Determine Saturation Limit\
+Author: Jay Pseudonym Cappucino \<jycappucino@gmail.com>\
+Status: Draft\
+Type: Process\
+Created: 2022-02-28\
+\
+&nbsp;

```

Comment:
The backslashes at the end of every line are going to cause some formatting problems and the `&nbsp;` sequences are bound to show up in the raw.  Keep in mind these CIPs appear in other contexts than Github and will have to move freely across platforms & get converted into other types of markup.  For the sake of portability there should be nothing here except for standard Markdown syntax.